src/main/java/nl/zandervdm/stayput/Commands/StayputCommand.java

```diff
@@ -30,16 +34,37 @@ public boolean onCommand(CommandSender commandSender, Command command, String s,
             }
             this.plugin.reloadConfig();
             this.plugin.setupConfig();
+            this.plugin.checkTableRebuild();
+            // Re-setup the dimensions if the config got altered.
+            this.plugin.setupDimensions();
             this.sendMessage(commandSender, "Config has been reloaded!");
             return true;
         }
 
+        if(executedCommand.equals("listworlds")) {
+            this.sendMessage(commandSender, "-- Dimension List --");
+            ImmutableMultimap<String,String> dimensions = this.plugin.getDimensionManager().getDimensions();
+            for(String dimension_name : dimensions.keySet() ) {
+                this.sendMessage(commandSender, dimension_name + ":");
+                dimensions.get(dimension_name).forEach(world_name -> {
+                    this.sendMessage(commandSender," - " + world_name);
+                });
+            }
+        }
+
+        if(executedCommand.equals("rebuildTables")) {
+            this.sendMessage(commandSender, "-- Deleting and Rebuilding Position --");
```

Comment:
Maybe add a permission check here? I assume not everyone should be allowed to run this command.